plugins/package-managers/gradle-plugin/src/main/kotlin/OrtModelBuilder.kt

```diff
@@ -173,6 +170,9 @@ internal class OrtModelBuilder : ToolingModelBuilder {
         }.filterIsInstance<ResolvedArtifactResult>()
     }
 
+    private fun AttributeContainer.getValueByName(name: String): Any? =
```

Comment:
Could be move out of class.

Reply:
I've added a commit that introduces a new file to bundle these extension functions.